Jetcaster/wear/src/main/java/com/example/jetcaster/ui/queue/QueueScreen.kt

```diff
@@ -201,34 +202,48 @@ fun ButtonsContent(
     enabled: Boolean = true,
     modifier: Modifier = Modifier,
 ) {
+    val interactionSource1 = remember { MutableInteractionSource() }
+    val interactionSource2 = remember { MutableInteractionSource() }
 
-    Row(
+    Box(
         modifier = modifier
             .padding(bottom = 16.dp)
             .height(52.dp),
-        verticalAlignment = Alignment.CenterVertically,
-        horizontalArrangement = Arrangement.spacedBy(6.dp, Alignment.CenterHorizontally),
+        contentAlignment = Alignment.Center,
     ) {
-        Button(
-            imageVector = Icons.Outlined.PlayArrow,
-            contentDescription = stringResource(id = R.string.button_play_content_description),
-            onClick = {
-                onPlayButtonClick()
-                onPlayEpisodes(episodes)
-            },
-            modifier = Modifier
-                .weight(weight = 0.3F, fill = false),
-            enabled = enabled,
-        )
-        Button(
-            imageVector = Icons.Outlined.Delete,
-            contentDescription =
-            stringResource(id = R.string.button_delete_queue_content_description),
-            onClick = onDeleteQueueEpisodes,
-            modifier = Modifier
-                .weight(weight = 0.3F, fill = false),
-            enabled = enabled,
-        )
+        ButtonGroup(Modifier.fillMaxWidth()) {
```

Comment:
check that this version on buttongroup isnt deprecated

Reply:
this is not deprecated on Wear Material 3